config/features.yml

```diff
@@ -1816,10 +1816,6 @@ features:
     actor_type: user
     description: Show updated confirmation page with form data & downloadable PDF
     enable_in_development: true
-  hlr_updateed_contnet:
```

Comment:
Reminder to remove this from the console [based on the instructions here ](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Removingold/unusedfeaturetoggles) after merge

Reply:
Thanks! I'll ask one of our backend engineers to help me out with that. cc: @dfong-adh & @kayline